app/models/listing.rb

```diff
@@ -1,4 +1,5 @@
 class Listing < ApplicationRecord
   has_many :images
   has_many :bookings, dependent: :destroy
+  has_many :reviews
```

Comment:
We have established in my previous reviews that Listing does not have a direct relationship with Review.
This is the reason why we removed the `listing_id` from the Reviews table.
No need for this here because you can just easily access Listing through Booking.